strawberryfields/apps/sample.py

```diff
@@ -57,22 +56,14 @@
 QUANTUM_BACKENDS = ("gaussian",)
 """tuple[str]: Available quantum backends for sampling."""
 
-BACKEND_DEFAULTS = {
-    "remote": False,
-    "backend": "gaussian",
-    "threshold": True,
-    "postselect": 0,
-}
+BACKEND_DEFAULTS = {"remote": False, "backend": "gaussian", "threshold": True, "postselect": 0}
```

Comment:
new black -l setting? :laughing:

Reply:
Yep, I had run black without a `-l` originally, the new one is `black -l 100` (although I do this through a pre-commit hook).